client/rest-high-level/src/main/java/org/elasticsearch/client/license/StartBasicRequest.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Licensed to Elasticsearch under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.elasticsearch.client.license;
+
+import org.elasticsearch.action.ActionRequestValidationException;
+import org.elasticsearch.action.support.master.AcknowledgedRequest;
+
+public class StartBasicRequest extends AcknowledgedRequest<StartBasicRequest> {
+    private final boolean acknowledge;
+
+    public StartBasicRequest() {
+        this(false);
+    }
+
+    public StartBasicRequest(boolean acknowledge) {
+        this.acknowledge = acknowledge;
+    }
+
+    @Override
+    public ActionRequestValidationException validate() {
```

Comment:
You can leave this out if you are not using it

Reply:
with ccbc485 `StartBasicRequest` extends `TimedRequest`, added couple test routines for that